packages/Stylers/src/helpers.js

```diff
@@ -27,10 +27,17 @@ export const lineHeight = (scale = 0) => `
 
 export const spacer = numSpaces => `${Number.parseInt(tokens.space, 10) * numSpaces}px`;
 
-export const z = level => {
+function zIsValid(level) {
   const validLevels = ["auto", "initial", "inherit", "-1", "0", "1", "2", "3", "4", "5", "6", "7"];
+  return validLevels.includes(level);
+}
+
+export const z = level => {
+  return zIsValid(level) ? `z-index: ${level};` : `z-index: 1;`;
+};
 
-  return validLevels.includes(level) ? `z-index: ${level};` : `z-index: 1;`;
+z.toNumber = level => {
```

Comment:
Clever, but do you think it's better to be more consistent with other helpers and go with `z()` and `zValue()`?

https://github.com/acl-services/paprika/blob/master/packages/Stylers/src/helpers.js#L9-L13
https://github.com/acl-services/paprika/blob/master/packages/Stylers/src/helpers.js#L18-L22

Reply:
🤔 Value doesn't describe that you are getting a number you could be getting `1px` `1vh` `1%` `1` or `{1}` right?